tests/garage/tf/algos/test_ddpg.py

```diff
@@ -62,7 +61,8 @@ def test_ddpg_pendulum(self):
         This environment has a [-3, 3] action_space bound.
         """
         with LocalTFRunner(snapshot_config, sess=self.sess) as runner:
-            env = normalize(GymEnv('InvertedPendulum-v2'))
+            env = normalize(
```

Comment:
Should we override `max_episode_length` with 100 instead of 1000 return by `env.spec.max_episode_length` calculated by `GymEnv`? If override yes, why 100?